File: src/main/java/net/internetisalie/lunar/lang/psi/LuaTokenType.java
package net.internetisalie.lunar.lang.psi;

import com.intellij.psi.tree.IElementType;
import net.internetisalie.lunar.lang.LuaLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class LuaTokenType extends IElementType {

    public LuaTokenType(@NotNull @NonNls String debugName) {
        super(debugName, LuaLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "LuaTokenType." + super.toString();
    }

}

File: src/main/kotlin/net/internetisalie/lunar/lang/psi/LuaTokenType.kt
package net.internetisalie.lunar.lang.psi

import com.intellij.psi.tree.IElementType
import net.internetisalie.lunar.lang.LuaLanguage

class LuaTokenType(debugName: String) : IElementType(debugName, LuaLanguage) {
    override fun toString(): String = "LuaTokenType." + super.toString()
}
